docs/FAQ/how-to-deploy-bold-bi-in-eks-using-application-load-balancer.md

```diff
@@ -0,0 +1,220 @@
+            BOLD_SERVICES_DB_USER: "boldbi@boldbi-docker"
+
+            # The database user's password
+            BOLD_SERVICES_DB_PASSWORD: "F8o:z$jasoKkel"
+
+            # If the database name is not specified, the system will create a new database called bold services.
+            # If you specify a database name, it should already exist on the server.
+            BOLD_SERVICES_DB_NAME: ""
+
+            # For PostgreSQL DB Servers, this is an optional parameter.
+            # The system will use the database name postgres by default.
+            # If your database server uses a different default database, please provide it here.
+            BOLD_SERVICES_POSTGRESQL_MAINTENANCE_DB: "postgres"
+
+            # If your database server requires additional connection string parameters, include them here
+            # Connection string parameters can be found in the official document.
+            # My SQL: https://dev.mysql.com/doc/connector-net/en/connector-net-8-0-connection-options.html
+            # PostgreSQL: https://www.npgsql.org/doc/connection-string-parameters.html
+            # MS SQL: https://docs.microsoft.com/en-us/dotnet/api/system.data.sqlclient.sqlconnection.connectionstring
+            # Note: A semicolon(;) should be used to separate multiple parameters.
+            BOLD_SERVICES_DB_ADDITIONAL_PARAMETERS: ""
+          ---
+        </details>
+           
+    * If you need to use the **Bing Map** widget feature, enter the value for `widget_bing_map_enable` environment variable as `true` and the API key value for `widget_bing_map_api_key`.
+       
+       ![Enable-Bingmap](../images/deploy/eks/bingmap_enable.png)
+
+11. If you have a DNS to map with the application, proceed to the next steps, otherwise, skip to **Step 13**. 
+
+12. Download the `ingress_alb.yaml` file from [here](https://raw.githubusercontent.com/boldbi/boldbi-kubernetes/main/deploy/deploy/ingress_alb.yaml) Uncomment the host value and replace your DNS hostname with `example.com` in ingress_alb.yaml file in line **18**.
+
+      ![DNS](../images/faq/ingress_alb_host.png)
+
+13. If you have the SSL certificate for your DNS and need to configure the site with it, follow these steps of section A or B, otherwise, skip to **Step 13**.
+
+    **A. SSL configuration using TLS secret:**
+	* Run the following command to create a TLS secret with your own SSL certificate.
+
+	```sh
+	kubectl create secret tls bold-tls -n bold-services --key <key-path> --cert <certificate-path>
+	```
+        
+	* Now, uncomment the `tls` section and replace your DNS hostname with `example.com` in ingress spec and save the file.
+
+	![ingress DNS](../images/faq/tls_ingress_alb.png)
+	
+    **B. SSL configuration using ACM ARN:**
+       * Create an ACM Certificate for your domain and verify ownership. Note down the Amazon Resource Name (ARN) of the Certificate you just created. You can find the ARN in the ACM Management Console, as shown below.
```

Comment:
* Create an ACM Certificate for your domain and verify ownership. Note down the Amazon Resource Name (ARN) of the Certificate you just created. Find the ARN in the ACM Management Console as follows.

Reply:
concern addressed